IPython/html/nbextensions.py

```diff
@@ -74,8 +77,10 @@ def _maybe_copy(src, dest, verbose=1):
     if _should_copy(src, dest, verbose):
         if verbose >= 1:
             print("copying %s -> %s" % (src, dest))
-        shutil.copy2(src, dest)
-
+        try:
+            shutil.copy2(src, dest)
+        except IOError as e:
+            print(str(e), file=sys.stderr)
```

Comment:
Yes. Exceptions tell the calling code that something is wrong, which may be able to do something about it. Additionally, if copying one file has failed:

1. It's very likely that copying the next one will fail, so for a large extension, the user will see a long line of intimidating errors.
2. It's very likely that the extension won't work correctly, so there's no use pretending to the caller that everything's fine.